espnet/nets/pytorch_backend/transducer/custom_layers.py

```diff
@@ -0,0 +1,222 @@
+"""Convolution-related modules definition for transformer-transducer."""
```

Comment:
Use more informative file name, for example, `conv_layers.py`?

Reply:
Or how about splitting this file into three, e.g., `vgg2l.py`, `causal_conv1d.py`, and `tdnn.py`. I think it is the best.

Reply:
agree